tests/test_integration.py

```
@@ -184,56 +195,56 @@
 
 
-def test_model_registry_operations(temp_mlflow_home):
+def test_model_registry_operations(temp_mlflow_home) -> None:
     """Test model registry operations."""
     model_name = "credit_default_test"
-    
+
     # Register a dummy model
     with mlflow.start_run():
         mlflow.sklearn.log_model(
             sk_model=DummyClassifier(),
```

Ruff (0.8.2)

Undefined name `DummyClassifier`

(F821)